setup.py

```diff
@@ -80,10 +80,10 @@
         'Programming Language :: Python',
         'Topic :: Internet :: WWW/HTTP :: Dynamic Content',
         'Topic :: Software Development :: Libraries :: Python Modules',
-        'Programming Language :: Python :: 2',
-        'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.7',
+        'Programming Language :: Python :: 3.8',
```

Comment:
After @diegodelemos pointing out that we don't really support python 3.8 we shouldn't add it to the classifiers.

Reply:
Example: https://github.com/inveniosoftware/invenio-access/pull/175/files#diff-354f30a63fb0907d4ad57269548329e3R20

Reply:
@ppanero @Glignos @diegodelemos then this should be reflected in the [RFC](https://github.com/inveniosoftware/rfcs/blob/master/framework/0016-python2.md) accordingly right?

Reply:
@topless yes we will update the RFC asap